README.md

```diff
@@ -0,0 +1,24 @@
+<h1 align="center"><img src="https://make.hmn.md/altis/Altis-logo.svg" width="89" alt="Altis" /> Local Chassis</h1>
+
+<p align="center">Local [Chassis](https://github.com/Chassis/Chassis) development server for <strong><a href="https://altis-dxp.com/">Altis</a></strong>.</p>
```

Comment:
Link needs to be HTML as you can't put Markdown inside HTML

Reply:
Cheers, sorted